docs/designs/2686-proposal-process.md

```diff
@@ -0,0 +1,90 @@
+# Proposing Changes to Tanzu Community Edition
+
+* Proposal: [https://github.com/vmware-tanzu/community-edition/issues/2686](https://github.com/vmware-tanzu/community-edition/issues/2686)
+
+## Introduction
+
+Tanzu Community Edition relies on design proposals to assess the viability of non-trivial changes
+to the project. This document covers details on the proposal process, which can also be used as
+guidelines for how to create a proposal.
+
+## The Proposal Process
+
+The proposal process is the process for reviewing a proposal and reaching a decision about whether
+to accept or decline the proposal.
+
+1. The proposal author [creates a brief issue](https://golang.org/issue/new) describing the proposal.
+
+   > A design doc is **not** required at this point.
+
+1. A discussion on the issue tracker aims to triage the proposal into one of three outcomes:
+
+    * Accepted (`proposal/accepted` label)
+    * Declined (`proposal/declined` label)
+    * Needs Design Doc (`proposal/needs-design-doc` label)
+
+   > If the proposal is Accepted or Declined, the process is done.
+
+1. Maintainers determine whether a design doc is required. If it is, a `proposal/needs-design-doc`
+   label.
+
+1. The proposal author creates a [design doc](#design-documents). This facilitates discussions
+   (in a GitHub PR) around the proposal.
+
+1. Maintainers review open proposals weekly to move them forward.
+
+## Design Documents
+
+When a design doc is requested, a proposal author should:
+
+1. Create a **branch** on their **fork** of the `community-edition` repository.
+
+1. Create the file `docs/designs/${GH_ISSUE_NUMBER}-${SHORT_PROPOSAL_TITLE}.md`.
```

Comment:
Will _all_ proposals, past and present, be stored here? Or will be at some latter time introduce a way for these to get trimmed? I'm ok with a long directory of proposals, but might get unruly.

Reply:
Unfortunately this will grow over time. As talked about in our review session, there _may_ be a future state where we move proposals out into it's own repository. But for the time being, we'll incubate this process in our own.